avocado/utils/disk.py

```diff
@@ -87,3 +94,41 @@ def get_filesystem_type(mount_point='/'):
             _, fs_file, fs_vfstype, _, _, _ = mount_line.split()
             if fs_file == mount_point:
                 return fs_vfstype
+
+
+def get_io_scheduler_list(device_name):
+    """
+    Returns io scheduler available for the IO Device
+    :param device_name: Device  name example like sda , hda
+    :return: list of IO scheduler
+    """
+    names = open(__sched_path(device_name)).read()
+    return names.translate(string.maketrans('[]', '  ')).split()
```

Comment:
What python version are you running? I tried this code with python 3.7 and got `AttributeError: module 'string' has no attribute 'maketrans'`. I also found this note on python release 3.1: _The string.maketrans() function is deprecated and is replaced by new static methods, bytes.maketrans() and bytearray.maketrans()._

Reply:
ah let me check my setup